src/lib/DepositViaPermit2Logic.sol

```diff
@@ -303,7 +303,7 @@ contract DepositViaPermit2Logic is DepositLogic {
                     // Derive the total memory offset for the witness.
                     let totalWitnessMemoryOffset :=
                         and(
-                            add(add(0x147, add(witness.length, iszero(iszero(witness.length)))), mul(compactCategory, 0x0b)),
+                            add(0x153, add(witness.length, iszero(iszero(witness.length)))),
```

Comment:
Looks like `forge fmt --check` is not liking this all on one line; gotta run `forge fmt` first